docs/user/dql/functions.rst

```diff
@@ -129,10 +129,23 @@ ADD
 Description
 >>>>>>>>>>>
 
-Specifications:
+Usage: add(x, y) calculates x plus y.
+
+Argument type: BYTE/SHORT/INTEGER/LONG/FLOAT/DOUBLE
 
-1. ADD(NUMBER T, NUMBER) -> T
+Return type: Wider type between types of x and y
 
+Synonyms: +
```

Comment:
This could be unclear...
Maybe: `add(x, y) is a complete equivalent of x + y`.

Reply:
I edited this to instead say `Addition Symbol (+)`, this should be more clear. I also updated it for all other functions implemented